static/beta/stage/navigation/settings-navigation.json

```diff
@@ -125,7 +125,7 @@
             "permissions": [
                 {
                     "method": "featureFlag",
-                    "args": ["platform.sources.integrations", false]
+                    "args": ["platform.sources.integrations", true]
```

Comment:
This should be `false` because it would show both `sources-integrations` and `integrations` in the nav resulting in double integrations in nav.